### PR TITLE
[knx] Add option to use CEMI frame format for serial devices.

### PR DESCRIPTION
**Pull request by holgerfriedrich**
CEMI frame format is used by newer serial devices like kBerry.
Calimero library can be configured to use CEMI instead of default EMI.

Followup on #10407. Relates to #4026.

Signed-off-by: Holger Friedrich <mail@holger-friedrich.de>

### CODE REVIEW

**Comment by holgerfriedrich** [comment]
@petero-dk #10407 was never merged to openhab-addons

File: bundles/org.openhab.binding.knx/src/main/resources/OH-INF/thing/serial.xml [hunk 1]
```diff
@@ -13,6 +13,13 @@
 				<label>Serial Port</label>
 				<description>The serial port to use for connecting to the KNX bus</description>
 			</parameter>
+			<parameter name="useCemi" type="boolean">
+				<label>Use CEMI</label>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.knx/src/main/resources/OH-INF/thing/serial.xml:17]: Please also update knx.properties
You can use the specific maven command.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.knx/src/main/resources/OH-INF/thing/serial.xml (hunk 1 above)
Comment (holgerfriedrich) [reply bundles/org.openhab.binding.knx/src/main/resources/OH-INF/thing/serial.xml:17]: @lolodomo Can you pls. give more details? I am not aware yet how to do this....

Comment (holgerfriedrich) [reply bundles/org.openhab.binding.knx/src/main/resources/OH-INF/thing/serial.xml:17]: got it, `mvn org.openhab.core.tools:i18n-maven-plugin:3.2.0:generate-default-translations -pl :org.openhab.binding.knx`

**Review by lolodomo — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.knx/src/main/resources/OH-INF/thing/serial.xml (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.knx/src/main/resources/OH-INF/thing/serial.xml:17]: Perfect

**Pull request merged**